consensus/hotstuff/notifications/telemetry.go

```diff
@@ -60,15 +50,65 @@ func (t *TelemetryConsumer) OnReceiveProposal(currentView uint64, proposal *mode
 		Uint64("block_view", block.View).
 		Hex("block_id", logging.ID(block.BlockID)).
 		Hex("block_proposer_id", logging.ID(block.ProposerID)).
-		Time("block_time", block.Timestamp)
-	if block.QC != nil {
+		Time("block_time", block.Timestamp).
+		Uint64("qc_block_view", block.QC.View).
```

Comment:
```suggestion
		Uint64("qc_view", block.QC.View).
```

Suggesting to change this here and elsewhere for consistency.

Noting that here, and for `OnReceiveQC`, this suggestion might be breaking for tools which analyze these logs, since this suggestion changes the name of an existing log field. However I'm pretty sure we're changing enough with Active Pacemaker that any tools we have would need to be modified anyway, so I think consolidating on a consistent field name here is OK.